checker-qual/src/main/java/org/checkerframework/framework/qual/DefaultQualifierInHierarchy.java

```diff
@@ -13,7 +13,8 @@
  *
  * <p>Other defaults can be specified for a checker via the {@link DefaultFor} meta-annotation,
  * which takes precedence over {@code DefaultQualifierInHierarchy}, or via {@code
- * GenericAnnotatedTypeFactory.addCheckedCodeDefaults()}.
+ * GenericAnnotatedTypeFactory.addCheckedCodeDefaults()}, CLIMB-to-top rule. For more details, see
+ * <a href="https://checkerframework.org/manual/#defaults">Default qualifier for the unannotated</a>
```

Comment:
```suggestion
 * GenericAnnotatedTypeFactory.addCheckedCodeDefaults()}.
 Also, the CLIMB-to-top rule applies unless explicitly overruled.
```

and then further below, next to a similar tag, add

```
@checker_framework.manual #defaults Default qualifiers for unannotated types
```